src/collections/_documentation/guides/getting-started/index.md

```diff
@@ -0,0 +1,118 @@
+
+- [Per-Project Integrations: Issue Notifications](https://docs.sentry.io/workflow/integrations/legacy-integrations/#issue-notifications)
+- [Routing Alerts to 3rd Party Integrations](https://docs.sentry.io/guides/alert-notifications/routing-alerts/#routing-alerts-to-3rd-party-integrations)
+
+### 3.2. Source Code Management
+
+By enabling Sentry's integration with your Source Code Management (SCM) provider, Sentry will:
+
+1. Identify **suspect commits** that likely introduced an error.
+2. **Suggest assignees** who can best resolve an error.
+3. Mark an issue as **Resolved via Commit or PR** to stop notifications about similar errors in past releases and notify you upon regression.
+4. Link a Sentry issue to a new or existing issue in your integrated [Issue Tracking](#23-issue-tracking) solution
+
+    ![Integrate SCMs]({% asset guides/getting-started/07.png @path %})
+
+Sentry has built-in integrations with: [GitHub](https://docs.sentry.io/workflow/integrations/global-integrations/#github), [GitHub Enterprise](https://docs.sentry.io/workflow/integrations/global-integrations/#github-enterprise), [GitLab](https://docs.sentry.io/workflow/integrations/global-integrations/#gitlab), and [Bitbucket](https://docs.sentry.io/workflow/integrations/global-integrations/#bitbucket). If you're using a different SCM provider or don't want Sentry to connect to your repository, checkout our [docs]((https://docs.sentry.io/workflow/releases/?platform=browser#alternatively-without-a-repository-integration)) to learn how you can still enable most of these features (#1 - #3) by telling Sentry about your commit metadata.
+
+### 3.3. Issue Tracking
+
+Enabling the 2-way integration with your Issue Tracking solution allows you to create a new issue from within the Sentry issue page, or link it to an existing one. In addition to **GitHub**, **GitLab**, and **Bitbucket** Issues that are enabled with the respective SCM integration, Sentry also integrates with: [Azure DevOps](https://docs.sentry.io/workflow/integrations/global-integrations/#azure-devops), [Clubhouse](https://docs.sentry.io/workflow/integrations/global-integrations/#clubhouse), [JIRA](https://docs.sentry.io/workflow/integrations/global-integrations/#jira), and others.
+
+> ![JIRA 2-Way Integration]({% asset guides/getting-started/jira-2way-binding.gif @path %})
+>
+> 2-Way integration with JIRA issue tracking.
+
+For a full list of all supported integrations, go to our documentation on [Integrations](https://docs.sentry.io/workflow/integrations/)
+
+## 4. Create Projects
+
+To start monitoring errors in your app with Sentry you'll need to initialize the SDK with a DSN key. To obtain a key, simply create a new Sentry project by going to `Settings > Projects > + Create Project`. Give the project a name and assign the responsible [Team/s](#3-set-up-teams). Now, any error that occurs in your code (wherever it may be deployed and running) will be associated with that specific project.
```

Comment:
```suggestion
To start monitoring errors in your app with Sentry, you'll need to initialize the SDK with a DSN key. To obtain a key, create a new Sentry project by going to `Settings > Projects > + Create Project`. Give the project a name and assign the responsible [Team/s](#3-set-up-teams). Now, any error that occurs in your code (wherever it may be deployed and running) will be associated with that specific project.
```